Kaushik kc/add cursor pointer on the cards

KaushikKC opened the pull request:
resolves #12 

Added Cursor pointer the whole card

KaushikKC committed:
corrected the detail page image size

KaushikKC committed:
Added cursor pointer to the card

bonface221 reviewed (approved):
This is great work @KaushikKC

bonface221 committed:
Delete yarn.lock

there were conflicts in the yarn.lock actually I'm thinking of adding it in the gitignore to prevent conflicts

bonface221 reviewed (approved):
Nice job
